maestro-ios/src/main/java/ios/idb/IdbIOSDevice.kt

```diff
@@ -418,6 +425,72 @@ class IdbIOSDevice(
         }
     }
 
+    // Warning: This method reads all bytes into memory. This can probably be optimized if necessary.
+    override fun startScreenRecording(out: Sink): Result<IOSScreenRecording, Throwable> {
+        val future = CompletableFuture<Void>()
```

Comment:
Can we use xcrun simct recordVideo command here directly?

Reply:
What is the benefit of that?

Reply:
Not depending on idb

Reply:
Let's address that holistically once we have a solution ready